modules/core/src/lib/attribute-transition-utils.js

```diff
@@ -4,13 +4,17 @@ import {padArray} from '../utils/array-utils';
 const noop = () => {};
 const DEFAULT_TRANSITION_SETTINGS = {
   type: 'interpolation',
-  duration: 0,
-  easing: t => t,
   onStart: noop,
   onEnd: noop,
   onInterrupt: noop,
   onUpdate: noop,
-  enter: x => x
+  enter: x => x,
+  // interpolation
+  duration: 0,
+  easing: t => t,
+  // spring
+  stiffness: 0.1,
+  damping: 0.5
```

Comment:
These values might be a bit snappy for defaults. If it was up to me, I might drop the `stiffness` to `0.05`. Though, if you tried a bunch of values and you preferred these, I'm okay with them! :-)